app/views/posts/_new.html.erb

```diff
@@ -0,0 +1,16 @@
+<!-- app/views/posts/new.html.erb -->
+<%= form_with(model: [@current_user, @current_user.posts.build], url: user_posts_path(@current_user)) do |form| %>
+  <div>
+    <%= form.label :title %>
+    <%= form.text_field :title %>
+  </div>
+
+  <div>
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+  </div>
+
+  <div>
+    <%= form.submit 'Create Post' %>
+  </div>
+<% end %>
```

Comment:
- [ ] It would be nice to add the link to the form to create a new post on the user's profile, this will make the form easier to access  to create a new post 👍🏿

Reply:
its Optional give me the link were it says like that on the project requirements